<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPanel</class>
 <widget class="QDialog" name="CPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Panel</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="installButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Install as Service</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uninstallButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uninstall Service</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to use WaterMate, 
 a local file search engine!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCopyright">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Please Email WaterLin.org@gmail.com for help!
 Product Homepage: 
http://en.waterlin.org/projects/WaterMate/index.html</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
